app/src/main/java/com/mapbox/maps/testapp/examples/snapshotter/DataDrivenMapSnapshotterActivity.kt

```diff
@@ -32,7 +32,7 @@ class DataDrivenMapSnapshotterActivity : AppCompatActivity() {
     super.onCreate(savedInstanceState)
     val snapshotMapOptions = MapSnapshotOptions.Builder()
       .size(Size(512.0f, 512.0f))
-      .resourceOptions(MapboxOptions.getDefaultResourceOptions(this))
+      .resourceOptions(MapboxMapOptions(this, resources.displayMetrics.density, null).resourceOptions)
```

Comment:
It shouldn't be necessary to create a `MapboxMapOptions` to get the `resourceOptions`.